Yafc/Windows/WelcomeScreen.cs

```diff
@@ -117,11 +117,17 @@ protected override void BuildContents(ImGui gui) {
         else if (errorMessage != null) {
             errorScroll.Build(gui);
             using (gui.EnterRow()) {
-                gui.BuildText("This error is critical. Unable to load project.");
+                string explanation = "YAFC was unable to load the project. You can disable the problematic mod once by clicking on 'Disable & reload' button, or you can disable it " +
+                                     "permanently for YAFC by copying the mod-folder, disabling the mod in the copy by editing mod-list.json, and pointing YAFC to the copy.";
```

Comment:
I don't think this should mention the "Disable & reload" button when that button isn't drawn.

Reply:
Fair. Will fix.

Reply:
Addressed in https://github.com/shpaass/yafc-ce/pull/351/commits/2a95be4ee9819201a3045b44188fbe72e0060811. I'll apply the new helper method to my previous refactor, in a separate commit.